PocketKit/Sources/Sync/RemoteMapping/Item+remoteMapping.swift

```diff
@@ -108,13 +104,13 @@ extension CDItem {
 
     func update(from corpusItem: CorpusSlateParts.Recommendation.CorpusItem, in space: Space) {
         givenURL = corpusItem.url
-        title = corpusItem.title
+        title = corpusItem.preview.title
         if let timeToRead = corpusItem.timeToRead {
             self.timeToRead = NSNumber(value: timeToRead)
         }
-        topImageURL = URL(string: corpusItem.imageUrl)
-        domain = corpusItem.publisher
-        excerpt = corpusItem.excerpt
+        topImageURL = URL(string: corpusItem.preview.image?.url ?? corpusItem.imageUrl)
```

Comment:
Is this because image is nullable?

I think we should not show topImage if preview is null. However, we can do that in a cleanup pass after this has been out for a while.

Reply:
okey